doc/safety/safety_requirements.rst

```diff
@@ -0,0 +1,110 @@
+
+Levels of requirements in the repository:
+=========================================
+* System Requirements:
+
+  System requirements describe the behaviour of the Zephyr RTOS (= the system here).
+  They describe the functionality of the Zephyr RTOS of a very high level point of view,
+  without going into details of the functionality itself.
+  The target of the system requirements is to get an overview of the included features of
+  the Zephyr RTOS.
+  In other words a person writing these requirements usually has some knowledge of the Zephyr RTOS
+  Project as the requirements are specific to an RTOS.
+
+* Software Requirements:
+
+  Software requirements refine the requirements of the system level at a more granular level so
+  that each requirement can be tested.
+  These requirements define the specific actions the feature shall be able to execute and the
+  behavior of the feature.
+
+UID Handling:
+=============
+For the requirements managment the tool `strictDoc
+<https://strictdoc.readthedocs.io/en/stable/>`__
+is used, therfore the UID handling shall be handled by the tool and this can be achived by Following
+handling:
+
+#. Don't add a UID filed for new requirements
+#. After finishing work on the new Requirements execute strictDoc manage auto-uid
+#. Add linking of the requirements with the new distributed UIDs
+
+After doing this the requirements are ready and a pull request can be created.
+The CI in the PR will check if the requirements UIDs are valid or if there are duplicates in it.
+If there are duplicates in the PR these need to be resolved by rebasing and re execute
+the steps above.
+
+Requirement Types:
+==================
+* Functional
+* Non-Functional
+
+Requirement title convention:
+=============================
+Use short and terse requirement titles.
+
+Pull Request requirement repository:
+
+* Adhere to the :ref:`contribute_to_zephyr` of the Zephyr Project.
+
+  * As long as they are applicable to the requirements repository.
+
+* Avoid creating larger commits that contain both trivial and non-trivial changes.
+
+* Avoid moving and changing requirements in the same commit.
+
+Characteristics of a good requirement:
+======================================
+* Unambiguous
+* Testable (verifiable)
+* Clear (concise, terse, simple, precise)
+* Correct
+* Understandable
+* Feasible (realistic, possible)
+* Independent
+* Atomic
+* Necessary
+* Implementation-free (abstract)
+
+Characteristics of a set of requirements:
+=========================================
+* Complete
+* Consistent
+* Non redundant
+
+.. note::
+   If you want to contribute to the requirements effort but don't know how to write requirements
+   or if you are new to writing requirements, we recommend to look into the
+   `EARS
+   <https://alistairmavin.com/ears/>`__ Syntax for writing requirements.
+   But we also accept contributions in other formats as long as the characteristics
+   of a requirement from above are met.
```

Comment:
Elevate note to a section:

Syntax 
====================
* Use of a recognized Requirements Syntax is recommended.
*  `EARS <https://alistairmavin.com/ears/>` is a good reference. 
    Particularly if you are unfamiliar with requirements writing.
* Other formats accepted as long as the characteristics
   of a requirement from above are met.

~~.. note::
   If you want to contribute to the requirements effort but don't know how to write requirements
   or if you are new to writing requirements, we recommend to look into the
   `EARS
   <https://alistairmavin.com/ears/>`__ Syntax for writing requirements.
   But we also accept contributions in other formats as long as the characteristics
   of a requirement from above are met.~~

Reply:
I could not figure out how to replace multiple lines.

Reply:
> I could not figure out how to replace multiple lines.

no problem I got your point here.